core/src/main/scala/com/rallyhealth/vapors/factfilter/data/Fact.scala

```diff
@@ -10,18 +10,29 @@ sealed abstract class Fact {
   val typeInfo: FactType[Value]
   val value: Value
 
-  override def toString: String = {
-    // TODO: JSON format for value? Show for value?
-    s"Fact(${typeInfo.fullName} = $value)"
+  // TODO: JSON format for value? Show for value?
```

Comment:
`Show` should have been called `Debug` if that makes sense. Another typeclass should be used for JSON representations.